### DISCUSSION TITLE
Open source SaaS App template with Appsmith & Pocketbase

### DISCUSSION THREAD

**Discussion by jakubsuchy** [pocketbase/pocketbase#4582] [Show and tell]
Hi All,
I wanted this for my own purpose but decided to publish as a template. I built a SaaS app using Appsmith with Pocketbase as a backend: https://github.com/jakubsuchy/saas-appsmith-pocketbase/tree/main

It supports:
- User registration, user login, authentication against Pocketbase
- Dynamic menu, logo configured from inside Pocketbase
- Email verification

Would love any comments, feedback, etc.!

**Comment by thuan2172001** [comment]
I think Appsmith is really fine if your use cases are simple. I have used Appsmith for 4 months in self-hosted mode. Honestly, I think it's fine for use cases related to table data and popups. On the other hand, it lags significantly after extended use with customized components and a medium-sized user base. Authorization headers and some features are limited in the free version. The theme options are not rich, and there are still many errors when using Appsmith
